python/examples/by-use-case/solana-usdc-yield-deposit/README.md

```diff
@@ -0,0 +1,66 @@
+# Solana USDC Yield Deposit Example
```

Comment:
Added a format for the readme, could you update it to follow the same structure as with the other examples?

Reply:
Sorry, totally forgot about the README